src/seismometer/controls/thresholds.py

```diff
@@ -0,0 +1,140 @@
+import logging
+
+import traitlets
+from ipywidgets import FloatSlider, Layout, ValueWidget, VBox
+
+from seismometer.plot.mpl._ux import alert_colors
+
+logger = logging.getLogger("seismometer")
+
+
+class PercentSliderListWidget(ValueWidget, VBox):
+    """
+    Vertical list of sliders
+    """
+
+    value = traitlets.Tuple(help="The selected values for the slider list")
+
+    def __init__(self, names: tuple[str], value: tuple[int] = None):
+        """A vertical list of sliders
+
+        Parameters
+        ----------
+        names : tuple[str]
+            Slider names
+        value : Optional[tuple[str]], optional
+            Slider start values, by default None, starts all sliders at zero.
+        """
+        super().__init__()
+        self.names = tuple(names)  # make immutable
+        self.value = value or tuple(0 for _ in names)  # set initial value
```

Comment:
Should initialization do more validation on passed values?  It's weird that untouched controls can have a state that is not possible to return to
![image](https://github.com/user-attachments/assets/8d8c450c-921d-4048-8041-a4f82e8d6994)
As soon as you touch it, does get pruned/winsored: ex (0.88, 1.0)

Reply:
Added initial clamping, and some validation around values being in the correct order (for Monotonic Sliders)